### ISSUE TITLE
[jaxrs-resteasy-eap]Build fail for JavaResteasyEapServerCodegen output

### ISSUE THREAD

**Issue by HugoMario** [swagger-api/swagger-codegen-generators#141]
Definition used: [petstore-with-fake-endpoints-models-for-testing](https://gist.github.com/HugoMario/5292d9936367ac145ba80e895793e003)

output:
```ERROR
swagger-codegen/samples/server/petstore/jaxrs-resteasy/eap-java8/src/main/java/io/swagger/api/impl/PetApiServiceImpl.java:[46,74] cannot find symbol
[ERROR]   symbol:   class FormDataContentDisposition
[ERROR]   location: class io.swagger.api.impl.PetApiServiceImpl

swagger-codegen/samples/server/petstore/jaxrs-resteasy/eap-java8/src/gen/java/io/swagger/api/PetApi.java:[104,32] cannot find symbol
[ERROR]   symbol:   class MultipartFormDataInput
[ERROR]   location: interface io.swagger.api.PetApi
```

**Comment by HugoMario** [comment]
Also present for `jaxrs-resteasy` which makes sense.

**Comment by HugoMario** [comment]
this is now fixed for `3.0.0` and ported to `2.4.0-SNAPSHOT`, so closing now.
